applications/OptimizationApplication/python_scripts/utilities/execution_policy_decorator.py

```diff
@@ -0,0 +1,88 @@
+import KratosMultiphysics as Kratos
```

Comment:
As I mentioned in my general comments, this the most important part of the analysiswrapper and it is not a simple utility. It must be moved to analysis_wrappers (execution_policies).